docs/language-server/protocol-language-server.md

```diff
@@ -372,6 +372,22 @@ interface Panic {
    */
   trace: ExpressionId[];
 }
+
+/**
+ * Indicates the expression is currently being computed. Optionally it
+ * provides description and percentate (`0.0-1.0`) of completeness.
+ */
+interface Pending {
```

Comment:
@4e6: In the case of `Pending` variant, `type` and `methodPointer` fields of `ExpressionUpdate` are not (currently) specified.

Right now neither `message` nor `progress` are specified.